twilio/common/schema.go

```diff
@@ -26,6 +26,11 @@ var (
 		Optional: true,
 	}
 
+	SchemaComputedSensitive = &options{
```

Comment:
Do you have ideas on how to figure out which properties are Sensitive?

Reply:
So far, any response property named `secret`.